packages/file_selector/file_selector/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
-## NEXT
+## 0.9.2+1
```

Comment:
This doesn't need a version bump, it's one of the [explicit excemptions](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version). The tooling just doesn't recognize it yet so it needs an override.